### PR TITLE
[stable8][upgrade] switch to debug logging on upgrade

### PR DESCRIPTION
**Pull request by MorrisJobke**
- resets afterwards
- adds output about the previous log level

Backport of #19441 
Approved in https://github.com/owncloud/core/pull/19441#issuecomment-144061483

cc @LukasReschke @nickvergessen @PVince81 @karlitschek

### CODE REVIEW

**Comment by PVince81** [comment]
:+1:

**Comment by karlitschek** [comment]
:+1:

File: core/ajax/update.php [hunk 1]
```diff
@@ -54,6 +54,12 @@
 		$eventSource->close();
 		OC_Config::setValue('maintenance', false);
 	});
+	$updater->listen('\OC\Updater', 'setDebugLogLevel', function ($logLevel, $logLevelName) use($eventSource, $l) {
+		$eventSource->send('success', (string)$l->t('Set log level to debug - current level: "%s"', [ $logLevelName ]));
```
Comment (nickvergessen) [inline core/ajax/update.php]: stable8 was 5.4 or 5.3 minimum? (Array Syntax)

Comment (LukasReschke) [inline core/ajax/update.php]: 5.4

Comment (LukasReschke) [inline core/ajax/update.php]: See https://github.com/owncloud/core/commit/dc9c5ecdbab5af860b8dce40918dd6b43a46825b

**Pull request merged**